pkg/storegateway/series_refs_test.go

```diff
@@ -176,3 +178,16 @@ func generateSeriesChunkRef(num int) []seriesChunkRef {
 
 	return out
 }
+
+func generateAggrChunk(num int) []storepb.AggrChunk {
```

Comment:
WDY about having this in series_chunks_test.go? That is where we should be dealing with AggrChunks

Reply:
Sure.